### DISCUSSION TITLE
Set Motion Detection Sensitivity

### DISCUSSION THREAD

**Discussion by ChrisHomeAssistant** [fuatakgun/eufy_security#954] [Q&A] [answered]
Hi!

Is there a way to control the motion detection sensitivity? I would like to create an automation to increase the sensitivity after sunset and reduce it again during daylight.

Best,
Christoph

**Comment by fuatakgun** [comment, marked as answer]
This already exists, if this exists on app but not on integration, create a feature request and share your debug sensor attributes over it

**Reply by ChrisHomeAssistant** [reply]
You are right, sorry, I just did not see the entity. Works perfectly fine.
Thanks
